### ISSUE TITLE
 how the parameters of the network were chosen？

### ISSUE THREAD

**Issue by chenjiefighting** [stonezwr/TSSL-BP#6]
First of all, the work of your team is wonderful, and I would like to apply the data set DVS-Gesture to your frame, and I would like to ask how the parameters of the network were chosen？

**Comment by stonezwr** [comment]
Hello

Thanks for your interest in our work.

In our most recent work, we test the DVS-Gesture, the config is:
Network:
  epochs: 400
  batch_size: 5
  n_steps: 300
  dataset: "DVS_Gesture"
  data_path: "../../datasets/DvsGesture"
  rule: "TSSLBP"
  lr: 0.0001
  loss: "count"
  n_class: 11
  desired_count: 180
  undesired_count: 30
  tau_m: 64
  tau_s: 8
  model: "LIF"

Layers:
  pooling_1:
    type: "pooling" # 32
    kernel_size: 4
    threshold: 1

  dropout_1:
    type: "dropout"
    p: 0.2

  conv_1:
    type: "conv" # 32
    in_channels: 2
    out_channels: 64
    kernel_size: 3
    weight_scale: 1
    padding: 1
    threshold: 1

  dropout_2:
    type: "dropout"
    p: 0.2

  conv_2:
    type: "conv" # 32
    in_channels: 64
    out_channels: 128
    kernel_size: 3
    weight_scale: 1
    padding: 1
    threshold: 1

  pooling_2:
    type: "pooling" # 16
    kernel_size: 2
    threshold: 1

  dropout_3:
    type: "dropout"
    p: 0.2

  conv_3:
    type: "conv" # 16
    in_channels: 128
    out_channels: 256
    kernel_size: 3
    weight_scale: 1
    padding: 1
    threshold: 1

  pooling_3: # 8
    type: "pooling"
    kernel_size: 2
    threshold: 1

  dropout_4:
    type: "dropout"
    p: 0.2

  hidden_1:
    type: "linear"
    n_inputs: 16384
    n_outputs: 256
    weight_scale: 1
    threshold: 1

  dropout_5:
    type: "dropout"
    p: 0.2

  output:
    type: "linear"
    n_inputs: 256
    n_outputs: 11
    weight_scale: 1
    threshold: 1

The performance we get is 95.49% (with some additional methods [1]). The paper is accepted by Neural Computation. The preprocessing follows the same steps in [2].

[1]Zhang, W., & Li, P. (2020). Skip-Connected Self-Recurrent Spiking Neural Networks with Joint Intrinsic Parameter and Synaptic Weight Training. arXiv preprint arXiv:2010.12691.
[2] Shrestha, S. B., & Orchard, G. (2018, December). SLAYER: spike layer error reassignment in time. In Proceedings of the 32nd International Conference on Neural Information Processing Systems (pp. 1419-1428).

Let me know if you have other questions.
